### DISCUSSION TITLE
Internalization and modification of draw.io Google Docs Add-on

### DISCUSSION THREAD

**Discussion by brnpl89** [jgraph/drawio#3974] [General]
We have tested the Google Docs add-on for draw.io and we found it very useful for our company. However, we want to internalize it in our own infrastructure and mofidy it in order to point it to our own self-hosted draw.io instance rather than the public one when we click on the "New Diagram" or "Edit Diagram" shortcuts or visualize the link for a diagram we inserted. Is it possible to do so?

**Comment by brnpl89** [comment]


**Reply by davidjgraph** [reply]
What's the "airbus instance"?
